frontend/src/components/GraphContent/GraphContent.js

```diff
@@ -3,20 +3,12 @@ import groupBy from 'lodash.groupby'
 import { StateCircle, TransitionSet, InitialStateArrow } from '/src/components'
 import { useProjectStore, useSelectionStore, useToolStore } from '/src/stores'
 import { locateTransition } from '/src/util/states'
-import { useStateDragging, useStateCreation, useTransitionCreation } from './hooks'
 
-const GraphContent = ({ containerRef }) => {
-  const tool = useToolStore(s => s.tool)
+const GraphContent = () => {
+  // const tool = useToolStore(s => s.tool)
```

Comment:
Please remove comments that do not directly document the code

Reply:
Please only review code relevant to yourself on the backend.